README.md

```diff
@@ -1,3 +1,19 @@
 # ImageQuiz
 
 See [the documentation website](http://jasig.github.io/ImageQuiz).
+
+## Building
+
+Build using 
+
+```shell_session
```

Comment:
Github doesn't recognize recognize `shell_session` as a language, these are the aliases that will trigger code highlights. https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3953-L3957